JS01-02.js

```diff
@@ -0,0 +1,10 @@
+var count = 0;
+while(count < 3){
+  var answer = prompt("日本の首都は？");
```

Comment:
`var` は昨今ではほとんど使われませんので
再代入する場合は `let`
再代入しない場合は `const`
をそれぞれ利用してほしいです。

```suggestion
  const answer = prompt("日本の首都は？");
```